data/automated_actions.xml

```diff
@@ -0,0 +1,14 @@
+<odoo>
+    <record id="bundle_create" model="base.automation">
+        <field name="name">Auto Add Followers on Invoice Sent</field>
+        <field name="model_id" ref="account.model_account_move"/>
+        <field name="trigger">on_write</field>
+        <field name="trigger_field_ids" eval="[(6, 0, [ref('account.field_account_move__is_move_sent')])]"/>
+        <field name="active" eval="True"/>
+        <field name="state">code</field>
+        <field name="code">
+users = env['res.users'].search([])
```

Comment:
How would you like those lucky users to be selected :D ?
Based on groups? Name? Id? Everything is fine to me, just need to know.
But you can already test it, it will add all users for now

Reply:
Can you create a new group "recouvrement"?

Reply:
Sure, done.
Created it as a standalone group, let me know if another group should imply that group or anything.